github/ci/prow-deploy/files/jobs/kubevirt/community/community-postsubmits.yaml

```diff
@@ -31,15 +30,9 @@ postsubmits:
                 mkdir -p $TARGET_DIR
                 pip3 install ruamel.yaml
                 ../project-infra/hack/git-pr.sh -c "python3 $(pwd)/hack/generate-devstats-repo-sql.py --repo-groups-sql $TARGET_DIR/repo_groups.sql" -p $REPO_DIR -o cncf -r devstats -b update-kubevirt-devstats-repo-sql
```

Comment:
As a heads up: there's an image that has git-pr.sh already baked in. It's not applicable here as we use the kubevirt/builder for a reason, but on other occasions it might be. Here's an example: https://github.com/kubevirt/project-infra/blob/1e25e450d01718139d6b68d6a739ac463d6e827e/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L127-L130